expression_prediction_evaluation.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<blockquote>computing R2 between predicted and ground truth data</blockquote><p>Kind of obvious to us what this means, but IMO this requires a link or a small explanation here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>